src/client/opamCommands.ml

```diff
@@ -2083,19 +2084,24 @@ let switch =
       | packages, _ -> packages
     in
     let compiler_packages rt ?repos switch compiler_opt =
+      let open OpamStd.Option.Op in
+      let single_lst p = (p >>| (fun x -> [x])) +! [] in
```

Comment:
Worth adding `OpamStd.Option.to_list` maybe ? :D
(which made its way into 4.08.0 IIRC)

Reply:
`Option.to_list` is indeed in 4.08.0

Reply:
A good occasion actually :)